common/reviews/api/rush-lib.api.md

```diff
@@ -922,13 +924,13 @@ export class _OperationMetadataManager {
     constructor(options: _IOperationMetadataManagerOptions);
     get relativeFilepaths(): string[];
     // (undocumented)
-    saveAsync({ durationInSeconds, cobuildContextId, cobuildRunnerId, logPath, errorLogPath }: _IOperationMetadata): Promise<void>;
+    saveAsync({ durationInSeconds, cobuildContextId, cobuildRunnerId, logPath, errorLogPath, logChunksPath }: _IOperationMetadata): Promise<void>;
```

Comment:
@dmichon-msft @iclanton It's out of scope for this PR, but Rush Stack should eliminate parameter destructuring from any public APIs in our code base.  By design, the compiler will emit into the .d.ts file the list of fields that get destructured as well as any local variable aliases, which is purely an implementation detail.  (The compiler maintainers tried to fix this problem, but later had to revert that fix due to obscure unsolvable edge cases.)  I'd actually like to create an ESLint rule to forbid this practice.